scss/base/_typography.scss

```diff
@@ -1,34 +1,34 @@
 @font-face {
   font-family: "Haas Grot Disp R";
-  src: url("/fonts/NeueHaasGrotDispRound-55Roman.otf");
+  src: url("/public/fonts/NeueHaasGrotDispRound-55Roman.otf");
   font-style: normal;
   font-weight: 500;
 }
 
 @font-face {
   font-family: "Haas Grot Disp R";
-  src: url("/fonts/NeueHaasGrotDispRound-65Medium.otf");
+  src: url("/public/fonts/NeueHaasGrotDispRound-65Medium.otf");
   font-style: normal;
   font-weight: 600;
 }
 
 @font-face {
   font-family: "Haas Grot Text R";
-  src: url("/fonts/NeueHaasGrotTextRound-55Roman.otf");
+  src: url("/public/fonts/NeueHaasGrotTextRound-55Roman.otf");
   font-style: normal;
   font-weight: 400;
 }
 
 @font-face {
   font-family: "Haas Grot Text R";
-  src: url("/fonts/NeueHaasGrotTextRound-65Medium.otf");
+  src: url("/public/fonts/NeueHaasGrotTextRound-65Medium.otf");
   font-style: normal;
   font-weight: 500;
 }
 
 @font-face {
   font-family: "Haas Grot Text R";
-  src: url("/fonts/NeueHaasGrotTextRound-75Bold.otf");
+  src: url("/public/fonts/NeueHaasGrotTextRound-75Bold.otf");
```

Comment:
does this break stuff for webb3? I think webb3 as a build script to copy these fonts around?

Reply:
it does not as far as i can tell. i had to add these here directly for the purposes of enabling the storybook preview within `compound-styles`

but, to be safe, i could separate these and / or copy the same build script from webb?